profiles/common/modules/custom/multisite_drupal_toolbox/multisite_drupal_toolbox.module

```diff
@@ -803,9 +806,24 @@ function _disable_block($module, $delta) {
 /**
  * Public function to add or delete nodes from the index.
  *
- * You need to add the module apachesolr as a dependancy to your feature.
+ * You need to add the module apachesolr as a dependency to your feature.
  */
 function multisite_drupal_toolbox_config_solr_bundle($bundle, $op) {
+  if (!module_exists('apachesolr')) {
+    watchdog(
+      'multisite_drupal_toolbox',
+      'Function %function was called with bundle %bundle and op %op, but the required apachesolr module is not enabled. You either need to add the module apachesolr as a dependency to the feature you are calling this function from, or prevent the call from happening if apachesolr is not a strict dependency and is disabled.',
```

Comment:
@tatarbj no there should not be a t() here at all. t() needs to be used e.g. where the entries of watchdog are printed on the screen by the dblog module, see for example https://api.drupal.org/api/drupal/modules!dblog!dblog.admin.inc/function/theme_dblog_message/7.x. If you use t() where you suggest it now, the log message will always show up in the current language when the message was logged (given there is a translation of course) instead of the current language when the message is displayed.